synapse/lib/stormlib/aha.py

```diff
@@ -161,10 +258,210 @@ async def _del(self, svcname):
         ),
         'storm': '''
             $pool = $lib.aha.pool.get($cmdopts.poolname)
-            if (not $pool) { $lib.exit(`No AHA serivce pool named: {$cmdopts.poolname}`) }
+            if (not $pool) { $lib.exit(`No AHA service pool named: {$cmdopts.poolname}`) }
 
             $pool.del($cmdopts.svcname)
             $lib.print(`AHA service ({$cmdopts.svcname}) removed from service pool ({$pool.name})`)
         ''',
     },
+    {
+        'name': 'aha.svc.stat',
+        'descr': '''Show all information for a specific aha service.
+
+If the --nexus argument is given, the Cortex will attempt to connect the service and report the Nexus offset of the service.
+
+The ready value indicates that a service has entered into the realtime change window for synchronizing changes from its leader.
+        ''',
+        'cmdargs': (
+            ('svc', {'help': 'The service to inspect.'}),
+            ('--nexus', {'help': 'Try to connect to online services and report their nexus offset.',
+                         'default': False, 'action': 'store_true'}),
+        ),
+        'storm': '''
+        function _getNexus(svcname) {
+            $_url = `aha://{$svcname}/`
```

Comment:
Should we construct a telepath URL dict rather than string manipulation?

Reply:
If we're using `$lib.telepath.open()` we're limited to generating a URL string.